lib/encodeCache.js

```diff
@@ -0,0 +1,39 @@
+import { encode as gpt3Encode, decode as gpt3Decode } from 'gpt-3-encoder';
+import { FastLRUCache } from './fastLruCache.js';
+
+class EncodeCache {
+    constructor() {
+        this.encodeCache = new FastLRUCache(1000);
+        this.decodeCache = new FastLRUCache(1000);
```

Comment:
trying to understand why we need two caches

Reply:
Responded in Slack, but to sum it up we don't really - we don't use decode that much in Cortex currently. . . and we certainly don't need to prepopulate it.  So I'm going to remove the pre-population and reduce the size, but keep it in as a free-ish optimization for the cases in which it applies.